assets/aml-benchmark/components/src/prompt_crafter/main.py

```diff
@@ -106,7 +112,8 @@ def main(
     :param n_shots: Number of shots to use for few-shot prompts.
     :param output_pattern: Pattern to use for output prompts.
     :param prompt_pattern: Pattern to use for prompts.
-    :param output_data: Path to jsonl with generated prompts.
+    :param output_file: Path to jsonl with generated prompts.
+    :param ground_truth_column_name: Ground truth column.
```

Comment:
Instead of `Ground truth column`, the param description should say `Ground truth column name`.